docker/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
+### 2019-09-19
+ - Added python3 and the pip3 version of zap to containers
```

Comment:
`to containers` could be removed, that's implied.
It could have `in preparation for Python 2 deprecation.`, as a warning.